### ISSUE TITLE
chore(ui): playwright tests pass in UI mode but not headless

### ISSUE THREAD

**Issue by andrewrisse** [defenseunicorns/leapfrogai#323]
### Describe what should be investigated or refactored
Our playwright tests are passing when run in UI mode but not headless. This is a problem when committing code and requires overriding Husky pre-checks with the --no-verify flag. It will also cause problems when the E2E tests are run in the pipeline.

**Comment by andrewrisse** [comment]
It seems this happens when there are too many conversations and you run the tests, likely to do with needing to scroll things into view.
If you run supabase db reset, the tests pass.

Potential solutions: 
1. Always clear the db before running
2. Increase the viewport size of the headless browser (is that even a thing)
3. Modify tests to scroll if they can't find an item

**Comment by andrewrisse** [comment]
I tried scrolling, it didn't work initially and I didn't think it was worth the time to mess with it more.

Default viewport in headless mode is a reasonable size, although it can be overriden.

I added a ```supabase db reset``` command to the  end of the playwright scripts (```npm run test:integration```) which will allow the tests to pass. I also made sure the chat tests cleanup their created threads when finished. This should help avoid this issue, although we will have to remember in future tests to do cleanup as well.

I tried doing a global teardown that deleted all messages, but not all tests could pass because they need to not have to scroll things into view and there were too many conversations. Having each test cleanup when it is done seems like the better option right now.

We can try to do the scrolling again when writing future tests and if it works, modify the others.
